src/reanimated2/js-reanimated/index.ts

```diff
@@ -43,7 +43,7 @@ export const _updatePropsJS = (updates, viewRef) => {
     );
 
     if (typeof viewRef._component.setNativeProps === 'function') {
-      viewRef._component.setNativeProps({ style: rawStyles });
+      setNativeProps(viewRef._component, rawStyles);
```

Comment:
Shouldn't this be viewRef.current._component?
I was trying to use with next.js and while debugging viewRef._component is always undefined
@piaskowyk

Reply:
Sorry, I thought I installed 2.2.4